sonic-slave-buster/Dockerfile.j2

```diff
@@ -90,7 +90,7 @@ RUN apt-get update && apt-get install -y \
         imagemagick \
         ghostscript \
         groff \
-        libpcre3-dev \
+        libpcre2-dev \
```

Comment:
why this change?

Reply:
libpcre2 is required by libyang2 when building buster. This dependency error is not seen in bullseye build, though. That's why the change only in buster.

dpkg-source: info: extracting libyang2 in libyang2-2.0.112
dpkg-source: info: unpacking libyang2_2.0.112.orig.tar.gz
dpkg-source: info: unpacking libyang2_2.0.112-6.debian.tar.xz
/sonic/src/libyang2/libyang2-2.0.112 /sonic/src/libyang2
dpkg-buildpackage: info: source package libyang2
dpkg-buildpackage: info: source version 2.0.112-6
dpkg-buildpackage: info: source distribution unstable
dpkg-buildpackage: info: source changed by Ondřej Surý <ondrej@debian.org>
 dpkg-source --before-build .
dpkg-buildpackage: info: host architecture amd64
dpkg-checkbuilddeps: error: Unmet build dependencies: libpcre2-dev
dpkg-buildpackage: warning: build dependencies/conflicts unsatisfied; aborting
dpkg-buildpackage: warning: (Use -d flag to override.)
make[1]: *** [Makefile:21: /sonic/target/debs/buster/libyang2_2.0.112-6_amd64.deb] Error 3
make[1]: Leaving directory '/sonic/src/libyang2'
[  FAIL LOG END  ] [ target/debs/buster/libyang2_2.0.112-6_amd64.deb ]
make: *** [slave.mk:493: target/debs/buster/libyang2_2.0.112-6_amd64.deb] Error 1
make: *** Waiting for unfinished jobs....
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.
Makefile.work:332: recipe for target 'buster' failed
make[1]: *** [buster] Error 2
make[1]: Leaving directory '/projects/csg_sonic2/sr407497/sonic_repos/community_0419/sonic-buildimage'
Makefile:30: recipe for target 'target/debs/buster/frr_8.2.2-sonic-0_amd64.deb' failed
make: *** [target/debs/buster/frr_8.2.2-sonic-0_amd64.deb] Error 2

Reply:
Btw, libpcre3 is already present at L#150 in the same file.